subworkflows/local/arg.nf

```diff
@@ -8,17 +8,23 @@ include { AMRFINDERPLUS_RUN                 }  from '../../modules/nf-core/amrfi
 include { DEEPARG_DOWNLOADDATA              }  from '../../modules/nf-core/deeparg/downloaddata/main'
 include { DEEPARG_PREDICT                   }  from '../../modules/nf-core/deeparg/predict/main'
 include { FARGENE                           }  from '../../modules/nf-core/fargene/main'
-include { HAMRONIZATION_ABRICATE            }  from '../../modules/nf-core/hamronization/abricate/main'
-include { HAMRONIZATION_RGI                 }  from '../../modules/nf-core/hamronization/rgi/main'
-include { HAMRONIZATION_DEEPARG             }  from '../../modules/nf-core/hamronization/deeparg/main'
-include { HAMRONIZATION_AMRFINDERPLUS       }  from '../../modules/nf-core/hamronization/amrfinderplus/main'
-include { HAMRONIZATION_FARGENE             }  from '../../modules/nf-core/hamronization/fargene/main'
-include { HAMRONIZATION_SUMMARIZE           }  from '../../modules/nf-core/hamronization/summarize/main'
 include { RGI_CARDANNOTATION                }  from '../../modules/nf-core/rgi/cardannotation/main'
 include { RGI_MAIN                          }  from '../../modules/nf-core/rgi/main/main'
 include { UNTAR as UNTAR_CARD               }  from '../../modules/nf-core/untar/main'
 include { TABIX_BGZIP as ARG_TABIX_BGZIP    }  from '../../modules/nf-core/tabix/bgzip/main'
 include { MERGE_TAXONOMY_HAMRONIZATION      }  from '../../modules/local/merge_taxonomy_hamronization'
+include { ARGNORM as ARGNORM_AMRFINDERPLUS  }  from '../../modules/nf-core/argnorm/main'
```

Comment:
If these take the output of hAMRonization as input, I would put them below the hAMRization modules